runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -168,13 +168,32 @@ default boolean containInScreen()
 	}
 
 	@ConfigItem(
-		keyName = "rememberScreenBounds",
-		name = "Remember client position",
-		description = "Save the position and size of the client after exiting",
-		position = 13
+			keyName = "rememberScreenBounds",
+			name = "Remember client position",
+			description = "Save the position and size of the client after exiting",
+			position = 13
```

Comment:
The double tabs on these lines should be singular, just step them back one indent so they line up with the other ConfigItems. The other two toggles below this need fixing too.